crates/router/tests/connectors/stax.rs

```diff
@@ -297,81 +454,127 @@ async fn should_sync_refund() {
 // Creates a payment with incorrect CVC.
 #[actix_web::test]
 async fn should_fail_payment_for_incorrect_cvc() {
-    let response = CONNECTOR
-        .make_payment(
-            Some(types::PaymentsAuthorizeData {
+    let customer_response = CONNECTOR
+        .create_connector_customer(customer_details(), get_default_payment_info(None, None))
+        .await
+        .expect("Authorize payment response");
+    let connector_customer_id = match customer_response.response.unwrap() {
+        PaymentsResponseData::ConnectorCustomerResponse {
+            connector_customer_id,
+        } => Some(connector_customer_id),
+        _ => None,
+    };
+
+    let token_response = CONNECTOR
+        .create_connector_pm_token(
+            Some(types::PaymentMethodTokenizationData {
                 payment_method_data: types::api::PaymentMethodData::Card(api::Card {
-                    card_cvc: Secret::new("12345".to_string()),
+                    card_number: cards::CardNumber::from_str("4111111111111111").unwrap(),
+                    card_exp_month: Secret::new("11".to_string()),
+                    card_exp_year: Secret::new("2027".to_string()),
+                    card_cvc: Secret::new("123456".to_string()),
                     ..utils::CCardType::default().0
                 }),
-                ..utils::PaymentAuthorizeType::default().0
+                browser_info: None,
             }),
-            get_default_payment_info(),
+            get_default_payment_info(connector_customer_id, None),
         )
         .await
-        .unwrap();
+        .expect("Authorize payment response");
     assert_eq!(
-        response.response.unwrap_err().message,
-        "Your card's security code is invalid.".to_string(),
+        token_response.response.unwrap_err().reason,
+        Some("{\"card_cvv\":[\"The card cvv may not be greater than 99999.\"]}".to_string()),
```

Comment:
You can use raw string literals here for readability.

```suggestion
        Some(r#"{"card_cvv":["The card cvv may not be greater than 99999."]}"#.to_string()),
```

Reply:
Done